packages/components/tabs/src/tab-group.scss

```diff
@@ -61,13 +61,17 @@
   }
 
   .fade-start {
-    background: linear-gradient(0deg, transparent, var(--_background));
+    background: inherit;
     inset-block-start: 0;
+    /* stylelint-disable-next-line color-no-hex */
+    mask-image: linear-gradient(0deg, transparent, #000);
```

Comment:
Replace `#000` with a variable, so you don't need to disable the stylelint rule everywhere?